src/containers/FormikForm/components/FormikAvailability.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+// @ts-ignore
+import { RadioButtonGroup } from '@ndla/ui';
+import { injectT, tType } from '@ndla/i18n';
+import styled from '@emotion/styled';
+import { FieldInputProps } from 'formik';
+
+export const StyledDiv = styled('div')`
+  section > div > input {
+    display: none;
+  }
+  section > div > label {
+    font-size: 20px;
```

Comment:
samsvarer dette med noe annen tekststørrelser? Har vi noen standarer fra ndla/ design-manualen som heller burde følges?

Reply:
Tar begge de css spørsmålene her: Denne css'en er en slags `hack` for en feil som oppstår et eller annet sted i css'en til `<RadioButtonGroup />` som gjør at de input feltene (selve sirklene i radio buttons) får `width:100%`. Det gjør at man kan hover'e over Alle med musa men så er det Student som blir merkert. Dette fikser den width feilen. Ingenting sånt skjer i designmanualen og jeg klarer ikke å gjenskape det andre steder enn her.

Reply:
Det er jo litt kjipt å hacke det til da, burde det heller fikses i komponenten?

Har du sett på hvordan aod gjør det i FormView, det er ikke noe han har løst der som du kan bruke? :)

Reply:
Så nå på åssen Anders har impementert det og ser at han fikk samme feil som meg (`width:100%`). Så det er noe med denne komponenten, men skjønner ikke hvorfor vi ikke får det i designmanualen

![image](https://user-images.githubusercontent.com/10486712/107031310-42f31a80-67b2-11eb-82f8-72d8e64b8857.png)

Reply:
Det den styled diven min gjør er å fjerne denne gule strekken